R/Exclusion_criteria.R

```diff
@@ -0,0 +1,60 @@
+####exclusion criteria####
+#1. discontinued parkruns#
+#reorder
+df_event <- readRDS(file = "data/clean/dt_events_ts.rds")
```

Comment:
Files shouldn't be read in from scripts in the "R" folder. This folder is for storing functions.